### PR TITLE
Back02 - Setting routes for getting datas

### PR DESCRIPTION
**Pull request by wuwachon**

### CODE REVIEW

File: controllers/item-controller.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+const { Category, Game, Item } = require('../models')
+
+const itemController = {
+  getItems: async (req, res, next) => {
+    try {
+      const items = await Item.findAll()
+      res.json({
+        status: 'success',
+        data: items.toJSON()
```
Comment (wd-David) [inline controllers/item-controller.js]: `items` is already an array of objects, `items.toJSON()` would show an error: `TypeError: items.toJSON is not a function`.

File: package.json [hunk 2]
```diff
@@ -4,17 +4,23 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "start": "node app.js",
-    "dev": "nodemon app.js",
+    "start": "cross-env NODE_ENV=development node app.js",
+    "dev": "cross-env NODE_ENV=development nodemon app.js",
     "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (wd-David) [inline package.json:11]: Suggest adding scripts like `db:migrate` or `db:seed` to specify the manual operation.

**Review by wd-David — CHANGES_REQUESTED** [review]
Suggest adding `README.md` for better project setup and workflow, e.g. what are included in `.env`.

File: controllers/item-controller.js (hunk 1 above)
Comment (wuwachon) [reply controllers/item-controller.js]: fixed!

Comment (wuwachon) [reply controllers/item-controller.js]: updated README.md and .env.example docs as well

File: package.json (hunk 2 above)
Comment (wuwachon) [reply package.json:11]: added "migrate" and "seed" in scripts
and updated it in README.md as well

**Review by wd-David — APPROVED** [review]
LGTM.

**Pull request merged**